src/libs/API.js

```diff
@@ -4,6 +4,20 @@ import * as Request from './Request';
 import * as SequentialQueue from './Network/SequentialQueue';
 import {version} from '../../package.json';
 
+/**
+ * Function for WRITE commands.
+ * All calls to API.write() will be persisted to disk as a JSON object.
+ * That object will have all the data needed to make the WRITE request later. In other words, we'll queue the WRITE
+ * request so that if the network is unavailable or the app is closed, we can send the WRITE request whenever the
+ * network comes back.
+ *
+ * @param {String} command - Name of API command to call
+ * @param {Object} apiCommandParameters - The object of parameters to send to the API
+ * @param {Object} onyxData  - Object containing errors, loading states, and optimistic UI data
+ * @param {Object} [onyxData.optimisticData] - An object of data that will be merged into Onyx before the request is made
+ * @param {Object} [onyxData.successData] - An object of data that will be merged into Onyx when the request succeeds.
+ * @param {Object} [onyxData.failureData] - An object of data that will be merged into Onyx when the request fails.
```

Comment:
```suggestion
 * @param {Object} [onyxData.optimisticData] - Onyx instructions that will be passed to Onyx.update() before the request is made
 * @param {Object} [onyxData.successData] - Onyx instructions that will be passed to Onyx.update() when the request succeeds.
 * @param {Object} [onyxData.failureData] - Onyx instructions that will be passed to Onyx.update() when the request fails.
```

The other methods can be updated with this too.